api/src/organizational-entities/application/api/OrganizationFeatureItem.js

```diff
@@ -0,0 +1,8 @@
+class OrganizationFeatureItem {
+  constructor({ name, params }) {
+    this.name = name;
+    this.params = params;
+  }
+}
+
+export { OrganizationFeatureItem };
```

Comment:
Quand nous avons un seul élément a exporter comme un modèle, on export directement l'élément que l'on exporte.

```suggestion
export class OrganizationFeatureItem {
  constructor({ name, params }) {
    this.name = name;
    this.params = params;
  }
}
```